README.md

```diff
@@ -2,7 +2,19 @@
 
 # Fairness in machine learning - Mitigation algorithms
 
-A Python package that implements a variety of algorithms that mitigate unfairness in supervised machine learning including:
+A Python package that implements a variety of algorithms that mitigate unfairness in supervised machine learning.
+
+## Current Release
+
+* The current stable release is available at [FairLearn v0.2.0](https://github.com/microsoft/fairlearn/tree/v0.2.0).
+
+* Our latest work differs substantively from version 0.2.0, please investigate this repo for further information. 
```

Comment:
What do you mean by investigate here? I'm a little confused.

Reply:
changed to please visit....